content/en/tracing/connect_logs_and_traces/php.md

```diff
@@ -19,6 +19,16 @@ further_reading:
 
 ## Automatic Trace ID injection
 
+PHP’s logging design prevents Datadog PHP tracing library to inject trace and span IDs into your logs automatically. See the section below to learn how to connect your PHP Logs and traces manually.
```

Comment:
I would just tweak this wording of the first sentence a bit to make it a bit more technically correct:

> Given that there are many different ways to implement logging in PHP and many implementations completely circumvent PHP's built-in error-logging API, the Datadog PHP tracing library cannot reliably inject trace and span ID's into logs automatically.

It doesn't have to be those words exactly, but something along those lines. :)